src/accordion/accordion.spec.ts

```diff
@@ -26,6 +24,13 @@ function expectOpenPanels(nativeEl: HTMLElement, openPanelsDef: boolean[]) {
   expect(result).toEqual(openPanelsDef);
 }
 
+function createTestComponent(html: string, TestComponent: any): ComponentFixture<any> {
```

Comment:
The proper signature should probably be sth like `createTestComponent<T>(html: string, TestComponent: T): ComponentFixture<T>` but this would require that we invoke it like `createTestComponent<TestComponent>(html, TestComponent);` which is mouthful....